docs/architecture/adr-061-inter-validator-set-messaging.md

```diff
@@ -0,0 +1,108 @@
+# ADR 061: Inter validator set messaging
+
+## Changelog
+
+- 2021-09-29: Initial version of the document
+
+## Context
+
+### Definitions
+
+For the needs of this document, we define the following types of nodes:
+
+- Validator nodes, which participate in the consensus protocol, verify and sign blocks,
+- Full nodes, which store all blocks from the blockchain.
```

Comment:
They are both store and blocks, but not all of them potentially, since Tendermint has the block pruning mechanism.

```suggestion
- Full nodes, which execute valid blocks.
```